docs/api/app.md

```diff
@@ -647,6 +647,17 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.unityLauncher.setBadgeCount(count)` _Ubuntu Unity_
```

Comment:
In this case `app.unityLauncher` should be renamed, so that this api could work for other desktop environments that support a counter badge (Not sure if there are any :wink:).
My first attempt was to expose `app.dock.setBadgeCount` which I removed because there **might** be code like this out there:
```js
if(app.dock !== undefined) { //Assume we are on MacOS
  app.dock.setBadgeText("Hello World"); //:bomb:
}
```